mlflow/models/evaluation/default_evaluator.py

```diff
@@ -627,6 +632,10 @@ def _log_model_explainability(self):
             # Shap evaluation might fail on some edge cases, e.g., unsupported input data values
             # or unsupported model on specific shap explainer. Catch exception to prevent it
             # breaking the whole `evaluate` function.
+
+            if not self.evaluator_config.get("ignore_exceptions", True):
+                raise e
+
```

Comment:
added explicit config argument to not bury exceptions - this is useful when trying to detect NaN errors

Reply:
`ignore_exceptions` might sound like "ignore any exceptions during the evaluation process", but that's not correct. Can we rename it?

Reply:
Sorry for merge the PR too quickly.
We can rename it to be `explainability_ignore_exceptions" .

Reply:
If we add this config, we should update `mlflow.evalute` docstring.

Reply:
Possibly, but I don't know what you would call it. Maybe `ignore_shap_explainer_exceptions`?

Reply:
We have other configs like `explainability_nsamples", we'd better keep the same format and make them have the same prefix

Reply:
ah sorry I just saw the replies